### PR TITLE
DUOLO-12802 Fix for Plan always show diff on duplocloud_plan_settings

### PR DESCRIPTION
**Pull request by Duplo-nikv**
## Overview

Fixed showing diff if no change
## Summary of changes

This PR does the following:

- made specified_metadata schema attribute optional along with computed and added diff suppress function
- ...

## Testing performed

- [ ] Using unit tests
- [ ] Manually, on my local system
- [ ✓] Manually, on a remote test system

## Describe any breaking changes

- ...

### CODE REVIEW

**Comment by zafarabbas** [comment]
Task linked: [DUPLO-12802 TF Plan always show diff on duplocloud_plan_settings [contextual]](https://app.clickup.com/t/8655600/DUPLO-12802)

File: duplocloud/resource_duplo_plan_settings.go [hunk 1]
```diff
@@ -66,14 +66,20 @@ func duploPlanSettingsSchema() map[string]*schema.Schema {
...
+			Type:             schema.TypeList,
+			Elem:             &schema.Schema{Type: schema.TypeString},
+			Computed:         true,
+			Optional:         true,
+			DiffSuppressFunc: diffSuppressSpecifiedMetadata, //if removed it notifies the change when there is no change
 		},
 	}
 }
 
+func diffSuppressSpecifiedMetadata(k, old, new string, d *schema.ResourceData) bool {
```
Comment (komalkadam-dc) [inline duplocloud/resource_duplo_plan_settings.go:79]: can we move it utils, as it can be used by other resources

**Review by komalkadam-dc — CHANGES_REQUESTED** [review]

File: duplocloud/resource_duplo_plan_settings.go (hunk 1 above)
Comment (Duplo-nikv) [reply duplocloud/resource_duplo_plan_settings.go]: done

**Review by komalkadam-dc — APPROVED** [review]

**Review by tahir-duplo — APPROVED** [review]
LGTM

**Pull request merged**